rts/Game/Camera/OverviewController.cpp

```diff
@@ -4,18 +4,25 @@
 #include "OverviewController.h"
 
 #include "Map/Ground.h"
-#include "Map/ReadMap.h"
 #include "Game/UI/MiniMap.h"
 #include "Game/UI/MouseHandler.h"
-#include "Sim/Misc/GlobalConstants.h"
 #include "System/Log/ILog.h"
 
+static float fit_cam_height_to_map(float map_x, float map_y, float fov) {
+    const auto extra_margin = 1.037;
+    const auto fov_coefficient = 1.0/math::tan(fov * math::DEG_TO_RAD) * extra_margin;
```

Comment:
Also mixes doubles and floats. `1.0f` should do the trick.

Reply:
probably also `1.037f` above since else you get a double in the calculation anyway?